js/index.js

```diff
@@ -40,3 +40,103 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Nav bar
+let nav = document.querySelectorAll('a')
+console.log(nav);
+for (let i =0; i < nav.length; i++) {
+  nav[i].textContent = siteContent["nav"][`nav-item-${i+1}`];
+  nav[i].style.color = 'green';
+}
+
+let navBar = document.querySelector('nav')
```

Comment:
You only need to use let if the variable is going to change later, const should be your default and only use let if you need to.

This includes if you are adding properties, you can still use const. So every single variable on this page should be a const.